plugins/woocommerce-admin/client/order-attribution/index.js

```diff
@@ -0,0 +1 @@
+export * from './order-attribution-install-banner';
```

Comment:
I believe this folder pretends to have more order-attributions stuff inside, or just the banner? If it's just the banner I'd suggest rename it to `plugins/woocommerce-admin/client/order-attribution-install-banner/...`

Reply:
Fixed in 30f059e3a9b473309aa9930e5ad308ac8659b46c.